kubernetes/samples/charts/ingress-per-domain/README.md

```diff
@@ -52,21 +52,39 @@ voyager:
   # stats port
   statsPort: 30315
 ```
+
+Sample `values.yaml` for the Nginx Ingress:
```

Comment:
Sample `values.yaml` -> Sample `values.yaml` file (make this edit for Voyager and Traefik, also)

Reply:
done